packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -33,13 +33,30 @@ const es2015 = {
     features: ['function "name" property'],
   },
   "transform-arrow-functions": {
-    features: ["arrow functions"],
+    features: [
+      "arrow functions / 0 parameters",
+      "arrow functions / 1 parameter, no brackets",
+      "arrow functions / multiple parameters",
+      'arrow functions / lexical "this" binding',
+      'arrow functions / "this" unchanged by call or apply',
+      "arrow functions / can't be bound, can be curried",
+      'arrow functions / lexical "arguments" binding',
+      "arrow functions / no line break between params and <code>=></code>",
+      "arrow functions / correct precedence",
+      'arrow functions / no "prototype" property',
+    ],
   },
   "transform-block-scoped-functions": {
     features: ["block-level function declaration"],
   },
   "transform-classes": {
-    features: ["class", "super"],
+    features: [
+      "class",
+      "super",
+      'arrow functions / lexical "super" binding in constructors',
+      'arrow functions / lexical "super" binding in methods',
+      'arrow functions / lexical "new.target" binding',
```

Comment:
We don't need the `new.target` one here, since it's compiled separately from classes.